### ISSUE TITLE
'Commit area' stuck after one commit

### ISSUE THREAD

**Issue by marvinde** [desktop/desktop#4359]
## Description

I have the message "Committing to %branch%" and the commit message and description are not empty when my commit is done, I have to restart GitHub Desktop to fix this.

## Version

* GitHub Desktop: 1.1.1

* Operating system: Microsoft Windows [version 10.0.16299.309]

## Steps to Reproduce

1. Change some files to a git repo
2. Select files to place in your commit, enter a message and a description and click on the commit button

### Expected Behavior

The "commit area" should be cleared (the commit message and description should be emptied and the commit button should be back to "Commit to %branch%").

### Actual Behavior

GitHub Desktop is processing the commit and when it's done the selected files are removed from the **changed files** list but the commit message and description are still there and the commit button is stuck on "Committing to %branch%"

### Logs

[2018-03-29.desktop.production.log](https://github.com/desktop/desktop/files/1860083/2018-03-29.desktop.production.log)

**Comment by shiftkey** [comment]
@marvinde thanks for the report and the logs! I don't see any mention of the `git commit` in the logs (I was hoping that we'd see an error to explain why it got stuck) - does the repository have any Git hooks enabled that might be running in the background when committing?

**Comment by shiftkey** [comment]
This also reminds me a bit of #4046:

> This seems to happen on occasions when my laptop is running slowly. Usually I've switched to another app while the commit is happening. But when I switch back, the Summary line is there.

Does this feel similar to your setup?

**Comment by marvinde** [comment]
@shiftkey Hey, sorry for taking a long time to answer.

I tried with a repository that doesn't have any Git hook and I had the same issue.
I did a clean reinstall with the last update (1.1.1) and I don't have the same problem anymore, the "commit area" is cleared, but sometimes I do have the same issue as described in #4046, when I switch to another app during the commit.

I close this issue since I no longer have the problem, it may have been fixed with 1.1.1
